web/src/components/molecules/Schema/FieldModal/index.tsx

```diff
@@ -402,7 +251,9 @@ const FieldCreationModal: React.FC<Props> = ({
                   {groups?.map(group => (
                     <Select.Option key={group.id} value={group.id}>
                       {group.name}{" "}
-                      <StyledGroupKey className="ant-form-item-extra">#{group.key}</StyledGroupKey>
+                      <span style={{ fontSize: 12, marginLeft: 4 }} className="ant-form-item-extra">
+                        #{group.key}
+                      </span>
```

Comment:
Remove inline styles.

```suggestion
                      <StyledGroupKey className="ant-form-item-extra">
                        #{group.key}
                      </span>
```